github-webhooks-to-s3/main.tf

```diff
@@ -0,0 +1,81 @@
+locals {
+  name = "${var.project}-${var.env}-${var.service}"
+
+  tags = {
+    managedBy = "terraform"
+    env       = "${var.env}"
+    owner     = "${var.owner}"
+    service   = "${var.service}"
+    owner     = "${var.owner}"
+  }
+}
+
+data "aws_iam_policy_document" "assume_role" {
+  statement {
+    sid     = "EnableLambdaAssumeRole"
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["lambda.amazonaws.com"]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "firehose-write" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "firehose:PutRecord",
+    ]
+
+    resources = ["${aws_kinesis_firehose_delivery_stream.firehose.arn}"]
+  }
+}
+
+resource "aws_iam_role_policy" "firehose" {
+  name   = "firehose"
+  role   = "${aws_iam_role.lambda.id}"
+  policy = "${data.aws_iam_policy_document.firehose-write.json}"
+}
+
+module "attach-logs" {
+  source    = "github.com/chanzuckerberg/cztack//aws-iam-policy-cwlogs?ref=v0.17.1"
```

Comment:
this can now be a local reference (instead of the full github path)

Reply:
can't include local references in modules...